dotnet/samples/README.md

```diff
@@ -0,0 +1,10 @@
+## Kernel Samples
+
+| Type                                                                        | Description                                                                                              |
+| --------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------- |
+| [`Getting started`](./GettingStarted/README.md)                             | A single step-by-step tutorial to get started                                                            |
+| [`Tutorials`](./Tutorials/README.md)                                        | More in depth step-by-step tutorials                                                                     |
+| [`Concepts`](./Concepts/README.md)                                          | Concepts by feature specific code snippets                                                               |
```

Comment:
`This section contains focussed samples which illustrate all of the concepts included in the Semantic Kernel. Look here to find a sample which demonstrate how to use all of the features of the Semantic Kernel.`

Reply:
Currently we don't have Demos that demonstrate `All` features. 
Updated to: `Look here to find a sample which demonstrate how to use many of Semantic Kernel features.`